### ISSUE TITLE
cba_settings Server Side

### ISSUE THREAD

**Issue by MitchJC93** [acemod/ACE3#6057]
**Arma 3 Version:** `1.80.143869` (stable)
**CBA Version:** `3.5.0.171204` (stable)
**ACE3 Version:** `3.12.1.31` (stable)

**Mods:**
```
- CBA_A3
- ace
-acex
- CUP
- 3cb
- RHS
```

**Description:**
Server Side ACE parameters not being recognised.

First off this could be a CBA issue rather than ACE but this has a knock on with ACE setup which you may not be aware of.

CBA_Settings.sqf - As per instructions, is to be placed in userconfig root with -filepatching enabled.
Server fails to boot - This is due to CBA_Settings.sqf not being found.

Fix - CBA_Settings.sqf actually needs to be placed in the Arma 3 Server root AND userconfig folder.

If it's just in Arma 3 Server there's issues with force force *ACE Parameter* being able to be overwritten by clients and also parameters not being transferred to clients. 

Examples being - 
*cba_settings.sqf on Server:

force force ace_nightvision_fogScaling = 0.3;
force force ace_nightvision_effectScaling = 0.3;

Server setting as viewed on client:
ace_nightvision_fogScaling = 1; (default and can be overwritten by client)
ace_nightvision_effectScaling = 1; (default and can be overwritten by client)

Long story short:
cba_settings.sqf in userconfig only - Server fails to boot (missing sqf)
cba_settings.sqf in Arma 3 Sever only - Server boots, some settings transferred, overwritten by client.
cba_settings.sqf in BOTH userconfig AND Arma 3 Sever - Happy days, works and can't be overwritten.

Another point to this is Advanced Fatigue has been broken since transferring to CBA. We've had to disable it due to there being next to no recovery and very high fatigue. I suspect this is related to this issue as changing cba_settings in either userconfig or Arma 3 Server can manipulate the fault. Will leave it with you if you decide to raise a separate issue for this one. 

**Where did the issue occur?**
- Dedicated

**Placed Modules:**
All ACE Modules removed since transfer to CBA.

**Comment by commy2** [comment]
>Fix - CBA_Settings.sqf actually needs to be placed in the Arma 3 Server root AND userconfig folder.
What do you mean by "cba_settings.sqf in userconfig only" and "cba_settings.sqf in Arma 3 Sever only"?

It needs to be placed inside the userconfig folder inside the game folder. E.g.:
```
<game folder>\arma3.exe
<game folder>\arma3server.exe
<game folder>\userconfig\cba_settings.sqf
```
And nowhere else.

**Comment by commy2** [comment]
You should upload some RPT files, because what you're describing doesn't sound right.

**Comment by MitchJC93** [comment]
This issue can be closed. No longer occurs since ACE 3.12.1.31.
